bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-simditor-mention",
-  "version": "4.0.0-alpha.1",
+  "version": "4.0.1",
```

Comment:
Alright! Yeah I was thinking that with the decaffeination, I'm not sure we could actually merge it back. We can always hold of decaffeination